lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloDriver.cs

```diff
@@ -55,10 +55,11 @@ public sealed class HelloDriver : IObserver<IAllocatedEvaluator>, IObserver<IEva
         private readonly REEFFileNames _fileNames;
 
         [Inject]
-        private HelloDriver(REEFFileNames fileNames)
+        private HelloDriver(REEFFileNames fileNames, IEvaluatorRequestor evaluatorRequestor)
         {
             _fileNames = fileNames;
             ClrHandlerHelper.GenerateClassHierarchy(GetGlobalAssemblies());
+            evaluatorRequestor.Submit(new EvaluatorRequest(number: 1, megaBytes: 64));
```

Comment:
I'm not sure this can be done in the constructor here. At least on the Java side, the guarantee we give is that any REEF API only works *after* `StartTime` fired. What's the semantics for C#?